content/api/cypress-api/custom-commands.md

```diff
@@ -411,6 +416,15 @@ You can also modify the behavior of existing Cypress commands. This is useful to
 always set some defaults to avoid creating another command that ends up using
 the original.
 
+<Alert type="warning">
+
+`Cypress.Commands.overwrite` can only overwrite commands, not queries. If you
+want to modify the behavior of a query, you'll need to use
+[`Cypress.Commands.overwriteQuery`](/api/cypress-api/custom-queries#overwriting-existing-queries)
+instead.
+
+</Alert>
+
```

Comment:
```suggestion
```

Not sure if this should be completely removed or if maybe there's some other advice we can offer if they're trying to overwrite a query?

Reply:
Updated to offer new advice about how to customize queries.

```
`Cypress.Commands.overwrite` can only overwrite commands, not queries. If you
want to modify the behavior of a query, you'll need to add your updated version
under a new name using
[`Cypress.Commands.addQuery`](/api/cypress-api/custom-queries) instead.
```